aiida_lammps/parsers/parse_raw/lammps_output.py

```diff
@@ -42,6 +41,9 @@ def parse_outputfile(
     if file_contents is not None:
         data = file_contents.split("\n")
 
+    for index, line in enumerate(data):
+        line = line.strip()
```

Comment:
This is not doing anything really

Reply:
Aha yes I think that is a remnant from a conflict resolution during a merge. Well spotted. Removing it now